README.md

```diff
@@ -270,7 +270,7 @@ A BibTeX entry for LaTeX users is
 ```
 
 SymPy is BSD licensed, so you are free to use it whatever you like, be
-it academic, commercial, creating forks or derivatives, as long as you
+it academic, commercial, creating forks, or derivatives, as long as you
```

Comment:
I think this was fine since "forks" and "derivatives" go with the verb "creating"